src/js/components/award/idv/activity/ActivityChartTooltip.jsx

```diff
@@ -372,7 +362,7 @@ export default class IdvActivityTooltip extends React.Component {
                                     {
                                         this.getLinks(
                                             'agency',
-                                            data.awardingAgencyId,
+                                            data.awardingAgencySlug || data.awardingAgencyId,
```

Comment:
Is there a case when we'll need 'data.awardingAgencyId' here?  It's slightly confusing because the getLinks function thinks it's getting a slug here.  I think if we don't have a slug we should not pass the awardingAgencyId.  We should consider not pass anything.  Then if the slug is an empty string, only show text instead of the link.

Reply:
You're right, thanks. If agency doesn't have a slug, don't provide a link.